### PR TITLE
Lab b5

### PR DESCRIPTION
**Pull request by TonyLianLong**

### CODE REVIEW

File: labs/b5.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    * Just `ping` the server once (we assume the internet connection is reliable and the packet will not be dropped)
+    * You can use `man ping` to see how to make the `ping` only ping the server once, and what the return value of `ping` command means. Use `if` to decide what to print.
+
+2. Write a shell script `mac.sh` which processes the output of `ip` command and displays the MAC address of the network interface `eth0` of your VM.
+    * First figure out how to use `ip` command to get an output which contains the information we want
+    * Then use `head` and `tail` command and pipes to tailor `ip`'s output to one line
+    * Use `cut` command ([Examples](https://www.geeksforgeeks.org/cut-command-linux-examples/)) to get the MAC address. Since we know that the MAC address has fixed length, feel free to count the indices.
+    * The final shell script only has to have one line, although a answer with multiple lines are also acceptable.
+
+
```
Comment (BernardZhao) [inline labs/b5.md]: Maybe add `## Submission` here so that the final line doesn't get skimmed over

File: labs/b5.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+It is undeniable that the internet is an important system that has redefined our world. The ability to develop networks and allow devices to communicate is critical to modern day computer systems.  This lab will take a look into the basics of computer networking and then examine networks through the perspective of a sysadmin.
+
+We will be using web browsing as an analogy to understand the basics of networking. What exactly happens when I go web browsing for cat pictures?
+
+But first let's take a short dive into the details of networking.
+
+--------------------------------------------------------------------------------
+
+# MAC
+Media access control (MAC) addresses are a identifiers uniquely assigned to network interfaces.
```
Comment (BernardZhao) [inline labs/b5.md]:
```suggestion
Media access control (MAC) addresses are identifiers uniquely assigned to network interfaces.
```

File: labs/b5.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+name = domain
+value = name of dns server for domain
+
+    This record points to another dns server that can provide an authoritative  answer for the domain. Think of this as redirecting you to another nameserver.
+3. **CNAME records**
+name = alias
+value = canonical name
+
+    These records point to the canonical name for a given alias for example docs.google.com would be an alias which simply points to documents.google.com
+try ww.facebook.com
```
Comment (BernardZhao) [inline labs/b5.md]:
```suggestion
try www.facebook.com
```

File: labs/b5.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+Service     | Port
+------------|------
+SSH         |22
+DNS         |53
+HTTP        |80
+HTTPS       |443
+
+
+# Sysadmin Commands
+As a sysadmin, trying to diagnose network issues can often be pretty challenging. Given the scale and complexity of networks, it's tough trying to narrow down the scope of a problem to a point of failure. What follows is a list of commands/tools that can help with triaging problems. There are a lot of tools and we don't expect you to memorize every single detail. However, it is important to know what tools exist and when to use them when a problems inevitably arise. If you ever need more details the `man` pages for these commands are a great place to turn to for reference.
```
Comment (BernardZhao) [inline labs/b5.md]:
```suggestion
As a sysadmin, trying to diagnose network issues can often be pretty challenging. Given the scale and complexity of networks, it's tough trying to narrow down the scope of a problem to a point of failure. What follows is a list of commands/tools that can help with triaging problems. There are a lot of tools and we don't expect you to memorize every single detail. However, it is important to know what tools exist and when to use them when problems inevitably arise. If you ever need more details the `man` pages for these commands are a great place to turn to for reference.
```

**Comment by TonyLianLong** [comment]
@BernardZhao @cg505 Any other comments?

**Pull request merged**